packages/widget/src/devMode/loadWidget.tsx

```diff
@@ -40,12 +40,6 @@ const DevMode = () => {
         }}>
           <Widget
             theme={theme}
-            settings={{
-              slippage: 5,
-            }}
-            routeConfig={{
-              goFast: true,
-            }}
```

Comment:
these props would cause the slippage/routePreference to always default to 5 and goFast: true in dev mode (more specifically, in http://localhost:5173) but it's unrelated to the bug. I figure we can just remove it so it doesn't confuse us later

Reply:
there's also a question here as to whether this is the intended behavior or if it should only initially default to slippage:5/goFast:true before the user changes the setting at which point itll just save what the user selects going forward?

Reply:
but if we do want to change that we can do it in a seperate PR

Reply:
this makes sense!

Reply:
I think this behavior is ok for now